.vscode/tasks.json

```diff
@@ -170,6 +170,30 @@
                     "${workspaceFolder}/src/test_driver/mbed/unit_tests/build"
                 ]
             }
+        },
+        {
+            "label": "Build specific EFR32 example",
+            "type": "shell",
+            "command": "scripts/run_in_build_env.sh",
+            "args": [
+                "./scripts/build/build_examples.py --target efr32-${input:efr32Board}-${input:efr32App}  build"
```

Comment:
Please  move this to the separate build rule that covers everyting

Reply:
Deleted that task. All boards can now be build with the Build specific example task.